src/mesh/RTT_Draco_Mesh_Reader.hh

```diff
@@ -0,0 +1,73 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   mesh/RTT_Draco_Mesh_Reader.hh
+ * \author Ryan Wollaeger <wollaeger@lanl.gov>
+ * \date   Friday, Jul 13, 2018, 08:38 am
+ * \brief  RTT_Draco_Mesh_Reader header file.
+ * \note   Copyright (C) 2018 Los Alamos National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#ifndef rtt_mesh_RTT_Draco_Mesh_Reader_hh
+#define rtt_mesh_RTT_Draco_Mesh_Reader_hh
+
+#include "Draco_Mesh_Reader.hh"
+#include "RTT_Format_Reader/RTT_Format_Reader.hh"
+
+namespace rtt_mesh {
+
+//===========================================================================//
+/*!
+ * \class RTT_Draco_Mesh_Reader
+ *
+ * \brief Wrap RTT file format reader and provide data to Draco_Mesh_Builder.
+ */
+//===========================================================================//
+
+class RTT_Draco_Mesh_Reader : public Draco_Mesh_Reader {
+private:
+  // >>> DATA
+
+  std::string filename;
+
+  std::shared_ptr<rtt_RTT_Format_Reader::RTT_Format_Reader> rtt_reader;
+
+public:
+  //! Constructor
+  explicit RTT_Draco_Mesh_Reader(std::string filename_);
+
+  // >>> SERVICES
+
+  DLL_PUBLIC_mesh void read_mesh();
+
+  // >>> ACCESSORS
+
+  unsigned get_numdim() const { return rtt_reader->get_dims_ndim(); }
+  unsigned get_numcells() const { return rtt_reader->get_dims_ncells(); }
```

Comment:
Consider promoting to `int64_t`? or will this break existing use of meshReader?
* numcells
* numnodes
* numsides, etc.

Reply:
Is this supposed to be `uint64_t` - this will be attempted shortly.

Reply:
Let's delay this change.  There are too many consequences to making these unsigned right now.  We can visit this later if it becomes important.

Reply:
Thank you for the input - this will be deferred...